### PR TITLE
Project: Responsive Design by Microverse [Collaborative project] (authors: Murilo Roque and Sayyod Saidov) 

### PR DESCRIPTION
**Pull request by MrSayyod**
- Created a new repository;
- created an extra branch;
- used float, flex and grid elements;
- set up stickler and linter files;
- committed changes often;
- HTML file checked by validator.w3.org;
- fixed stickler and stylelint errors.

### CODE REVIEW

File: styles/style.css [hunk 1]
```diff
@@ -0,0 +1,1103 @@
...
+.bg-image3 {
+  background: url(https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/03/Verily-coronavirus-screening-hed-796x419.jpg);
+}
+
+.bg-image1,
+.bg-image2,
+.bg-image3 {
+  height: 13rem;
+
+  /* width: 40.9rem; */
```
Comment (euqueme) [inline styles/style.css]: - [x] Delete line #91 as no commented code should exist at this point, you can always check previous versions of your code going through your past commits

File: styles/style.css [hunk 2]
```diff
@@ -0,0 +1,1103 @@
...
+  background-image: url(https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/03/cluno-growth-516x270.png);
+  background-position: 50%;
+  background-size: cover;
+}
+
+/* .div1-container-latest-news img {
+  height: 7.5rem;
+  width: 7.5rem;
+  margin-left: 1rem;
+  margin-right: 1rem; */
```
Comment (euqueme) [inline styles/style.css]: - [x] Delete this commented class

File: styles/style.css [hunk 3]
```diff
@@ -0,0 +1,1103 @@
...
+}
+
+.none {
+  display: none;
+}
+
+/* .categories, .cat-articles {
+  width: 100%;
+  height: auto;
+} */
```
Comment (euqueme) [inline styles/style.css]: - [x] Delete these commented classes

File: index.html [hunk 4]
```diff
@@ -0,0 +1,609 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TNW</title>
+    <link rel="stylesheet" href="styles/reset.css">
+    <link rel="stylesheet" href="styles/style.css">
+    <link rel="icon" href="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/icons/favicon-194x194.png?v=1583748939">
+</head>
+
+<body>
+   <header>
+     <div class="navbar">
```
Comment (euqueme) [inline index.html]: [OPTIONAL]

- [x] I strongly recommend changing the `<div>` tag on line 15 to be a `<nav>` tag

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @MrSayyod and @MuriloRoque :balloon: :lollipop: 

### Wow! your project looks impeccable :open_mouth:  your page looks almost like the original and you should be very proud of your work :muscle: :sparkler: there is still some work to do :construction_worker_man:  :construction_worker_man: so let's work on it :point_down: 

## Small and Medium Size Screens

#### Main articles

- [x] The main articles should be bigger in height for this screen size just like the original
- Small
  - Yours
image

  - Original
image

- Medium
  - Yours
image

-Original

image

### Latest Fundings

- [x] Only 3 articles should be displayed in this screen size you currently have 4

- Small
  - Yours
image

  - Original
image

- Medium
  - Yours
image

  - Original
image

- Medium Screens

#### navbar

- [x] The TNW logo and the white navbar row should be smaller in height

- Yours
image

- Original
image

#### Latest funding

- [ ] The navigation links should be display in a row in this screen size like the original

- Yours
image

- Original
image

## Big Screens

#### Main articles

- [x] The biggest article should have the TNW white letter logo at the leftmost corner on the top, and the date in the rightmost corner on the top, also all the articles should be slightly bigger in height

- Yours
image

- Original
image

### Latest fundings

- [x] The navigation links should be reversed in order for this screen size

- Yours
image

- Original
image

## Section Specific

#### CSS bad practices

- [x] No repeating of the same rule, here is a list of all the selectors you have repeated several times, let's chain define them or declare a class that we can use everywhere is needed in the HTML to save lines in our CSS and keep the code DRY (Don't Repeat Yourself)

- width: 100%; (11 times)
- z-index: 2; (2 times)
- display: flex; (15 times)
- justify-content: space-between; (3 times)
- display: none; (3 times)
- text-align: center; (3 times)
- height: 1.6rem; (3 times)
- font-size: 1.4rem; (3 times)
- color: #000; (4 times)
- font-weight: 400; (6 times)
- background-position: 50%; (33 times)
- background-size: cover; (33 times)
- color: #fff; (3 times)
- margin: 0; (18 times)
- font-size: 2rem; (4 times)
- font-weight: 700; (3 times)
- color: #f42; (8 times)
- flex-direction: column; (4 times)
- height: 7.5rem; (4 times)
- height: 7.5rem; (3 times)
- padding: 0; (3 times)
- font-size: 14px; (4 times)
- color: #aaa; (6 times)
- align-items: center; (4 times)
- color: #555; (3 times)
- text-decoration: none; (3 times)

### That's it team! :raised_hands: 

### Please :pray: do the changes :man_technologist: :man_technologist: and submit back for a code review when you are done :memo: 

Happy Coding! :rocket: 

Any question? ask me on slack **@ Maru** :woman_technologist: 

PROJECT STATUS: REQUESTED CHANGES :vertical_traffic_light:

File: styles/style.css (hunk 1 above)
Comment (MrSayyod) [reply styles/style.css]: Done

File: styles/style.css (hunk 2 above)
Comment (MrSayyod) [reply styles/style.css]: Done

File: styles/style.css (hunk 3 above)
Comment (MrSayyod) [reply styles/style.css]: Done

File: index.html (hunk 4 above)
Comment (MrSayyod) [reply index.html]: Done

**Comment by lekegitrepo** [comment]
### Hi Team 👏 👏 🥇 
### Good job 💯 for implementing all the required changes mentioned by the previous TSE.
### This project is approved.

**_However, I think you should try to fix the excessive margin on the right side of the page and try to make it consistent in all the breakpoints because it affects some images in some of the sections._**

### Reviewed by: Temitayo Adeleke

-   Github: [@lekegitrepo](https://github.com/lekegitrepo)
-   Twitter: [@leketemi](https://twitter.com/leketemi)
-   Linkedin: [linkedin](https://www.linkedin.com/in/adeleke-temitayo-a69125188/)

**Review by lekegitrepo — APPROVED** [review]
### Status: Approved.

**Pull request merged**